pkg/common/client.go

```diff
@@ -125,6 +127,31 @@ func newTLSConfig(clientCert, clientKey, caCert string) (*tls.Config, error) {
 	return config, nil
 }
 
+func MakeAdminClient(clientID string, kafkaAuthCfg *utils.KafkaAuthConfig, bootstrapServers []string) (sarama.ClusterAdmin, error) {
```

Comment:
Seems bad to have a function in `pkg/common/` take in arguments which are defined in `pkg/channel/consolidated` - can we move those structs up into common as well (probably in a new auth package or something)  ?

Reply:
OK, point taken :-) 

I will move